tests/ExcelMapper/ExcelColumnIndexAttributeTests.cs

```diff
@@ -17,7 +17,26 @@ public void Ctor_Int(int index)
         [Fact]
         public void Ctor_InvalidIndex_ThrowsArgumentOutOfRangeException()
         {
-            Assert.Throws<ArgumentOutOfRangeException>("index", () => new ExcelColumnIndexAttribute(-1));
+            Assert.Throws<ArgumentOutOfRangeException>("value", () => new ExcelColumnIndexAttribute(-1));
```

Comment:
this should still be "index" - could you add back the validation in the constructor